signal/signal.go

```diff
@@ -7,23 +7,23 @@ package signal
 
 const (
 	// StackTrace represents a command to print stack trace.
-	StackTrace = byte(0x1)
```

Comment:
Please keep these as they are. I don't want the reordering here break the clients compiled with a different version in an incompatible way.